### PR TITLE
Ft 51 ability to collapse expand token groups 564

### PR DESCRIPTION
**Pull request by MightyDevKnight**
- [x] Functional review @six7

### CODE REVIEW

**Review by six7 — CHANGES_REQUESTED** [review]

File: src/app/components/TokenSetListOrTree/TokenSetListOrTree.tsx [hunk 1]
```diff
@@ -55,13 +55,14 @@ export function TokenSetListOrTree<T extends TreeOrListItem>({
...
-              {(!item.isLeaf && displayType === 'tree') && (
+              {(!item.isLeaf && displayType === 'tree')
+              && (
                 <StyledFolderButton
                   type="button"
                   css={{ left: `${5 * item.level}px` }}
                   onClick={onToggleCollapsed}
                 >
-                  <StyledFolderButtonChevronBox collapsed={collapsed.includes(item.key)}>
+                  <StyledFolderButtonChevronBox>
```
[the remark this answers is not in the document]
Comment (MightyDevKnight) [reply src/app/components/TokenSetListOrTree/TokenSetListOrTree.tsx]: Done.

File: src/utils/tokensToTree.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { SingleToken } from '@/types/tokens';
```
Comment (six7) [inline src/utils/tokensToTree.ts:1]: Where is that file used?

**Review by six7 — CHANGES_REQUESTED** [review]
I think tokensToTree is unused currently?

I added a bit of styling fixes. Functionality is approved!

File: src/utils/tokensToTree.ts (hunk 2 above)
Comment (six7) [reply src/utils/tokensToTree.ts:1]: @MightyDevKnight also make sure that test coverage isnt failing

**Pull request closed without merging**